src/App.tsx

```diff
@@ -10,33 +11,19 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [tab, setTab] = React.useState(tabs[0]);
```

Comment:
```suggestion
  const [selectedTab, setSelectedTab] = React.useState(tabs[0]);
```
Use proper naming for variables - it always helps when you know what is inside without searching declaration.